### PR TITLE
Blockbase: Remove .nocomments fix

### PR DESCRIPTION
**Pull request by mikachan**
#### Changes proposed in this Pull Request:
Now that https://github.com/WordPress/gutenberg/pull/35743 has been merged, I believe we can remove the rule for hiding the `.nocomments` container.

To test, make sure the 'comments are closed' message still doesn't appear if there are no comments and comments are closed. (With the GB PR, this container won't render at all.)

### CODE REVIEW

**Review by MaggieCabrera — APPROVED** [review]
LGTM, let's move it to the queue of do not merge PRs :D

**Comment by pbking** [comment]
Note: Waiting for Gutenberg 9.x to land on wpcom simple before merging.

**Pull request merged**
